packages/spindle-ui/src/BottomButton/BottomButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactNode } from 'react';
+
+type Position = 'fixed' | 'sticky';
+
+type Props = {
+  position?: Position;
+  children: ReactNode;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+const BLOCK_NAME = 'spui-BottomButton';
+
+export const BottomButton: React.FC<Props> = ({
+  children,
+  className,
+  position = 'fixed',
+  ...rest
+}: Props) => {
+  const classnames = [BLOCK_NAME, `${BLOCK_NAME}--${position}`, className]
+    .filter(Boolean)
+    .join(' ');
```

Comment:
```suggestion
  const classnames = `${BLOCK_NAME} ${BLOCK_NAME}--${position} ${className}`
```

IMO. このパターンだとfilterの処理はなくても（変数でなくても）良さそうと思いました！今後の拡張性の考慮などでしょうか？

Reply:
他のコンポーネントでの記述に合わせてたというのと、className指定がない場合に空白ができない形になっていたからというのもあります…！